blog/20131031-website-redesign.md

```diff
@@ -25,17 +25,17 @@ generator.
 
 Anyone can submit a new blog post, corrections or translations to the MATE
 website by creating a free account on [GitHub](Markdown), forking the 
-[mate-desktop.org repository](http://github.com/mate-desktop/mate-desktop.org)
+[mate-desktop.org repository](https://github.com/mate-desktop/mate-desktop.org)
 and [creating a pull-request](https://help.github.com/articles/creating-a-pull-request).
 Pull-requests for mate-desktop.org will be reviewed by our Webmasters,
 merged and published here. The are some
-[other benefits of using a static site generator](http://getnikola.com/handbook.html#why-static).
+[other benefits of using a static site generator](https://getnikola.com/handbook.html#why-static).
 
 Unfortunately, we have decided not to migrate comments from the old
 website. From this point forward each new blog post will be linked to 
-a topic in the [MATE forums](http://forums.mate-desktop.org/). We are
+a topic in the [MATE forums](https://forums.mate-desktop.org/). We are
 doing this because we already have a
-[dedicated team of forum moderators](http://forums.mate-desktop.org/memberlist.php?mode=leaders)
+[dedicated team of forum moderators](https://forums.mate-desktop.org/memberlist.php?mode=leaders)
```

Comment:
forums.mate-desktop.org with/without https aren't working, error 522, connection timed out

Reply:
I will remove the the links as the forum is archived.

Reply:
Done